.classpath

```diff
@@ -0,0 +1,6 @@
+<?xml version="1.0" encoding="UTF-8"?>
```

Comment:
Why this?

Reply:
no idea buildship doing buildship things

Reply:
Sadness. More files that will be touched after a refresh gradle projects. -
Should we push these?

Reply:
i have pushed all if seen correct